test/e2e/e2e_suite_test.go

```diff
@@ -109,16 +109,15 @@ func dumpLogs() {
 	for _, component := range []string{"mic", "nmi"} {
 		podList := pod.List(pod.ListInput{
 			Lister:    kubeClient,
-			Namespace: corev1.NamespaceDefault,
+			Namespace: framework.NamespaceKubeSystem,
 			Labels: map[string]string{
 				"app.kubernetes.io/component": component,
 			},
 		})
 
 		for _, pod := range podList.Items {
 			By(fmt.Sprintf("Dumping logs for %s scheduled to %s", pod.Name, pod.Spec.NodeName))
-			_, err := exec.KubectlLogs(kubeconfigPath, pod.Name, corev1.NamespaceDefault)
-			Expect(err).To(BeNil())
+			_, _ = exec.KubectlLogs(kubeconfigPath, pod.Name, framework.NamespaceKubeSystem)
```

Comment:
Why are we removing the check here?

Reply:
I wanted to make sure the entire `AfterSuite` is run despite potential errors during `dumpLogs`, that's why I ignore the error here. But I realized I could wrap all the cleanup logic in a defer function. Let me include the change in a follow-up PR (I merged this PR before seeing this comment).

Reply:
I piggy ride the change in #734.